etc/sysctl.d/anonymizer-config-gateway.conf

```diff
@@ -33,4 +33,4 @@ net.ipv6.conf.all.forwarding = 0
 ## You can verify the setting applied by:
 ## cat /proc/sys/net/ipv6/conf/all/disable_ipv6
 ## which should return 1.
-net.ipv6.conf.all.disable_ipv6 = 1
+net.ipv6.conf.all.disable_ipv6 = 0
```

Comment:
Some of the comments above this need to be changed in order to be accurate - in particular `disable IPv6` is no longer correct, and `cat /proc/sys/net/ipv6/conf/all/disable_ipv6` will now return 0.

Reply:
Thanks, updated